app/js/SafeExec.js

```diff
@@ -28,12 +27,14 @@ function safeExec(command, options, callback) {
 
   let killTimer
 
-  const cleanup = _.once(function(err) {
+  let cleanedUp = 0
```

Comment:
I feel like this is less clear (for humans) than `_.once`, so while it is a bit overkill to load all of underscore for this one thing, I'm not sure we should sacrifice readability. What about using https://www.npmjs.com/package/lodash.once ?

Reply:
Agree. I rather put the logic in a local function, than using this 294 lines package. https://github.com/lodash/lodash/blob/4.1.1-npm-packages/lodash.once/index.js

Reply:
That is perhaps a surprising amount of code... but TBH I'd rather have 294 lines of upstream code, complete with tests and documentation, etc., than 30 lines of code (and tests) and consequent discussion in this PR that we have to maintain. I would still prefer to use a library here.

Reply:
Done in https://github.com/overleaf/filestore/pull/96/commits/ecdaff14f0647e8a7e7ec9f1775b472749ef4ca7